test/evergreen/run_format_configs.sh

```diff
@@ -41,7 +41,15 @@ wait_for_process()
 			ps $process > /dev/null; ps_exit_status=$?
 			if [ ${ps_exit_status} -eq "1" ] ; then
 				# The process is completed so remove the process id from the list of processes.
-				PID_LIST=(${PID_LIST[@]/$process})
+				# Need to use a loop to prevent partial regex matches.
+				unset NEW_PID_LIST
+				declare -a NEW_PID_LIST
+				for target in ${PID_LIST[@]};do
```

Comment:
Thanks for figuring out the bug in the script. Another approach to fix without using another array.

`for index in "${!PID_LIST[@]}"; do
  if [[ "${PID_LIST[$index]}" == "${process}" ]]; then
    unset PID_LIST[$index]
  fi
done`